extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -215,26 +243,14 @@ describe.configure().retryOnSaucelabs().run('integration amp-bind', function() {
       });
     });
 
-    it('should NOT change src when new value is a blocked URL', () => {
-      const button = fixture.doc.getElementById('disallowedVidUrlButton');
-      const vid = fixture.doc.getElementById('video');
-      expect(vid.getAttribute('src')).to
-          .equal('https://www.google.com/unbound.webm');;
-      button.click();
-      return waitForBindApplication().then(() => {
-        expect(vid.getAttribute('src')).to
-            .equal('https://www.google.com/unbound.webm');
-      });
-    });
```

Comment:
I'm not sure. Might have been a badly-resolved merge conflict? I've reverted this.